### DISCUSSION TITLE
Chia Transaction Lost

### DISCUSSION THREAD

**Discussion by BuzKillington** [Chia-Network/chia-blockchain#8212] [Support]
Hello, I farm in a pool, and I went to transfer by balance to my gate.io account (as I've done before). It shows the transaction in Chia app and the explorer, but nothing in my wallet, nor even a record that it's pending, etc. It's not a lot of money, but I want to figure this out so it doesn't happen again.  It's been 14 hours since I initiated the transaction. Thanks much

**Comment by loppefaaret** [comment]
> It shows the transaction in Chia app

where does it show the transaction in the app, if not in the wallet ? 
if it is listed on a blockexplorer, the transaction has gone through - there are some reports of gate.io having some issues - perhaps that is what you are experiencing ?

**Comment by BrandtH22** [comment]
@BuzKillington , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
